bossdata/spec.py

```diff
@@ -518,7 +522,6 @@ def get_valid_data(self, exposure_index=None, camera=None, pixel_quality_mask=No
             first_index = float(get_fiducial_pixel_index(10.0**hdu['loglam'][0]))
             if abs(first_index - round(first_index)) > 0.01:
                 raise RuntimeError('Wavelength grid not aligned with fiducial grid.')
-            first_index = int(round(first_index))
```

Comment:
@dkirkby I think I was testing something and removed it by mistake. Should I add it back in and submit a new PR?